src/Orleans.Serialization/Codecs/IntegerCodec.cs

```diff
@@ -898,7 +898,7 @@ internal static class SignedBinaryIntegerCodec<T> where T : struct, IBinaryInteg
             var byteCount = value.GetByteCount();
             writer.WriteFieldHeader(fieldIdDelta, expectedType, typeof(T), WireType.LengthPrefixed);
             writer.WriteVarUInt32((uint)byteCount);
-            Span<byte> bytes = byteCount < 64 ? stackalloc byte[byteCount] : new byte[byteCount];
```

Comment:
I just realized, this is the wrong Slice usage, it should be `.Slice(0, byteCount)`

Reply:
Ups, sorry (my fault).